bundle/compliance/policy/kube_api/data_adapter.rego

```diff
@@ -35,10 +35,12 @@ is_service_account_or_pod = pod
 
 is_service_account_or_pod = service_account
 
-containers = c {
+containers := c {
 	input.resource.kind == "Pod"
-	container_types := {"containers", "initContainers"}
-	c := pod.spec[container_types[t]]
+	c := {
+		"containers": object.get(pod.spec, "containers", {}),
```

Comment:
A bit confusing to have an access to `containers.containers`.

From my understanding, Kubernetes has these types of containers for a pod:
Application containers
Init containers
Ephemeral containers

We need to decide if we want to evaluate `ephemeral containers` as well.

I think that we can have if it makes sense:
`{ app_containers, init_containers, ephemeral containers }`